tasks/patches/add-sbom-and-push.yaml

```
@@ -102,6 +102,9 @@
     name: inject-sbom-and-push
     resources: {}
     script: |
+      # Expose base image digests
+      buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' >$(results.BASE_IMAGES_DIGESTS.path)
```

```suggestion
      buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' > $(results.BASE_IMAGES_DIGESTS.path)
```

simple readability suggestion